wasp/src/main/java/com/orhanobut/wasp/CertificateUtils.java

```diff
@@ -0,0 +1,102 @@
+package com.orhanobut.wasp;
+
+import android.content.Context;
+
+import com.squareup.okhttp.OkHttpClient;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.security.KeyStore;
+import java.security.cert.CertificateException;
+import java.security.cert.X509Certificate;
+
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.TrustManager;
+import javax.net.ssl.TrustManagerFactory;
+import javax.net.ssl.X509TrustManager;
+
+/**
+ * @author Emmar Kardeslik
+ */
+public final class CertificateUtils {
+
+    /**
+     * Hidden constructor to prevent instantiation.
+     */
+    private CertificateUtils() {
+    }
+
+    public static OkHttpClient getTrustAllCertHttpClient() {
+        try {
+            final TrustManager[] trustAllCerts = new TrustManager[]{
+                    new X509TrustManager() {
+                        @Override
+                        public void checkClientTrusted(X509Certificate[] chain, String authType)
+                                throws CertificateException {
+                        }
+
+                        @Override
+                        public void checkServerTrusted(X509Certificate[] chain, String authType)
+                                throws CertificateException {
+                        }
+
+                        @Override
+                        public X509Certificate[] getAcceptedIssuers() {
+                            return new X509Certificate[]{};
+                        }
+                    }
+            };
+
+            SSLContext sslContext = SSLContext.getInstance("TLS");
+            sslContext.init(null, trustAllCerts, new java.security.SecureRandom());
+
+            OkHttpClient okHttpClient = new OkHttpClient();
+            okHttpClient.setSslSocketFactory(sslContext.getSocketFactory());
+
+            return okHttpClient;
+        } catch (Exception e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public static OkHttpClient getPinnedCertHttpClient(Context context, int rawResId, String password) {
+
```

Comment:
I would remove the space for the consistency even though you love it :) rest is great